installation-and-configuration/configure-winrm-authentication.md

```diff
@@ -0,0 +1,29 @@
+---
+description: >-
+  This article explains how to configure WinRm authentication on
+  your machine to successfully run snapshots.
+---
+
+# Configure Windows Remote Management (WinRM)
+
+In order to successfully collect data, **SysKit Trace** will create a **remote PowerShell session** to the Security & Compliance Center.
+
+To make sure that collection can work properly, Windows Remote Management (WinRM) on your computer needs to allow **Basic authentication** (it's enabled by default). 
```

Comment:
perhaps add something about modern authentication somewhere in the article and how it's still required to use basic auth.

Reply:
Paragraph is added explaining why its required even with MFA